include/macis/gf/bandlan.hpp

```diff
@@ -200,53 +195,48 @@ void MyBandLan(const Functor &H,
       int band_indx_j = true_indx[jt];
 #pragma omp parallel for
       for(size_t coeff = 0; coeff < temp.size(); coeff++)
-        // temp[coeff] -= bandH[it - 1][jt - 1] * qs[band_indx_j][coeff];
-        temp[coeff] -= bandH[it - 1][jt - 1] * qs[N * band_indx_j + coeff];
+        temp[coeff] -=
+            bandH[(it - 1) * nLanIts + jt - 1] * qs[N * band_indx_j + coeff];
```

Comment:
Is there a reason you chose (what looks like to me) row-major for `bandH`?

Reply:
Probably just the similarity with how I was linearizing `qs` (where each vector is stored contiguously). In the case of `bandH`, since it's a Hermitian matrix, it does not really matter. But I can change it if you prefer to have all matrices in column-major.

Reply:
All depends on what is the most efficient storage scheme - i.e. which scheme will most likely give you the most opportunities for contiguous data access. It doesn't look like it particularly matters here.

However, it looks like this operation can be made a bit more efficient - correct me if I'm wrong, but your computing something that looks like 
```
temp(:) -= alpha * qs(:,band_idx_j)
```
where `alpha = bandH(it-1, jt-1)`. That's an [`axpy`](https://github.com/icl-utk-edu/blaspp/blob/master/include/blas/axpy.hh). Looks like it can be replaced on line [207](https://github.com/wavefunction91/MACIS/pull/13/files#diff-9e97d12887f4f89e090eaaea4234aa18f8ef8783e51b9e57109b3010a955572cR207) too

Reply:
Good catch!